servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategy.java

```diff
@@ -67,7 +72,9 @@
      * {@link Executor}.
      * @param handler {@link StreamingHttpService} to wrap.
      * @return Wrapped {@link StreamingHttpService}.
+     * @deprecated Will be replaced with a new class {@code StreamingHttpServiceToOffloadedStreamingHttpService}.
```

Comment:
Because this class is not available now, let's leave the same suggestion as for other methods:
```
 * @deprecated This method will be removed in future releases. If you depend on it, copy the implementation from
 * {@link DefaultHttpExecutionStrategy#offloadService(Executor, StreamingHttpService)}.
```